aom/src/main/java/com/nedap/archie/rules/Operator.java

```diff
@@ -39,6 +41,26 @@ public Expression getRightOperand() {
         return operands.size() > 1 ? operands.get(1) : null;
     }
 
+    public String getSymbol() {
+        return symbol;
+    }
+
+    public void setSymbol(String symbol) {
+        this.symbol = symbol;
+    }
+
+    public OperatorDef getOperatorDef() {
+        return operator == null ? null : new OperatorDefBuiltin(operator.getIdentifier());
+    }
+
+    public void setOperatorDef(OperatorDef operatorDef) {
+        if(operatorDef != null) {
+            if(operatorDef.getIdentifier() != null) {
+                operator = OperatorKind.parseFromIdentifier(operatorDef.getIdentifier());
+            }
+        }
```

Comment:
no, it should not set it to null, as the json can also contain `operator` separately. So it should do neither and the current behaviour is correct. Operator will be null if not otherwise set anyways.